arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-rollup-chain.md

```diff
@@ -0,0 +1,238 @@
+
+Here are the parameters you can use with `prepareChainConfig`:
+
+| Parameter              | Description                                                                          |
+|:------------------------|:-------------------------------------------------------------------------------------|
+|`chainId`            | Your Orbit chain's unique identifier. It differentiates your chain from others in the ecosystem. |
+|`DataAvailabilityCommittee`            |    Set to `false`, this boolean makes your chain as a Rollup, set to `true` configures it as an Anytrust chain. |
+|`InitialChainOwner`          |  Identifies who owns and controls the chain. |
+|`MaxCodeSize  `|   Sets the maximum size for contract bytecodes on the Orbit chain. e.g. Ethereum mainnet has a limit of 24,576 Bytes. |
+|`MaxInitCodeSize`    |   Similar to `MaxCodeSize`, defines the maximum size for your Orbit chain's **initialization** code. e.g. Ethereum mainnet limit is 49,152 Bytes. |
+
+
+Below is an example of how to use `prepareChainConfig` to set up a Rollup chain with a specific `chainId`, an `InitialChainOwner` (named as `deployer_address`):
+
+```js
+import { prepareChainConfig } from '@arbitrum/orbit-sdk';
+
+const chainConfig = prepareChainConfig({
+    chainId: Some_Chain_ID,
+    arbitrum: { InitialChainOwner: deployer_address, DataAvailabilityCommittee: false },
+});
+```
+
+### Rollup Configuration Parameters
+
+In this section, we'll provide detailed explanations of the various chain configuration parameters used in the deployment of Orbit chains. Understanding these parameters is critical to customizing your Orbit chain to suit your needs.
+
+
+| Parameter              | Description                                                                          |
+|:------------------------|:-------------------------------------------------------------------------------------|
+|`batchPoster`            | Sets the batch poster address of your Orbit chain. The batch poster account batches and compresses transactions on the Orbit chain and transmits them back to the parent chain. |
+|`validators`            |  Array of <a data-quicklook-from="validator">validator</a> addresses. Validators are responsible for validating the chain state and posting Rollup Blocks (`RBlocks`) back to the parent chain. They also monitor the chain and initiate challenges against potentially faulty RBlocks submitted by other validators. |
+|`nativeToken`          |  Determines the token used for paying gas fees on the Orbit chain. It can be set to `ETH` for regular chains or to any `ERC-20` token for **gas fee token network** Orbit chains. |
+|`confirmPeriodBlocks`  |   Sets the challenge period in terms of blocks, which is the time allowed for validators to dispute or challenge state assertions. On Arbitrum One and Arbitrum Nova, this is currently set to approximately seven days in block count. |
```

Comment:
We should mention that this is measured in L1 blocks.

Reply:
We should also add a recommended value: 45818 blocks

Reply:
Merged manually.

Reply:
Sorry, this is for `confirmPeriodBlocks`, not `batchPoster` .

Reply:
I could've seen it for myself, though...But I didn't. Thanks!

Reply:
Are the changes I made correct?